dask_elk/elk_entities/node.py

```diff
@@ -33,20 +33,29 @@ def __init__(self):
     def nodes(self):
         return self.__nodes
 
-    def get_nodes_from_elastic(self, elk_client):
+    def get_nodes_from_elastic(self, elk_client, wan_only):
```

Comment:
I would prefer if you set this with default value to `False`  So it won't break any call to `get_nodes_from_elastic` without passing `wan_only` parameter.